blockchain/config.go

```diff
@@ -143,6 +145,26 @@ func (cfg *Config) ProducerPrivateKey() crypto.PrivateKey {
 	return sk
 }
 
+// IsDeployerWhitelisted returns whether the address is whitelisted for replay transaction
+func (cfg *Config) IsDeployerWhitelisted(deployer address.Address) bool {
+	for _, v := range cfg.ReplayDeployerWhitelist {
+		if v[:3] == "io1" {
```

Comment:
It's better to use a function to parse string to address, supporting ethAddr and ioAddr. Some similar code lies in ioctl/util/util.go:157.

Reply:
fixed